ui/admin-portal/src/app/components/job/jobs/jobs.component.html

```diff
@@ -91,10 +91,10 @@
               <i class="fa-solid fa-briefcase" title="Job"></i>
             </a>
           </td>
-          <td title="Contact {{fullUserName(opp.contactUser)}}">
```

Comment:
Added a tooltip to display the full (non truncated) stage. Moved the contact tooltip from the stage to the created date column. Was there a reason it was on the stage? Also what is the purpose of this, as i didn't there was no contact user on my test job opp so it just said 'contact'.